Source/SelfService/Web/package.json

```diff
@@ -32,13 +32,16 @@
         "date-fns": "^2.28.0",
         "generate-password-browser": "^1.1.0",
         "notistack": "2.0.4",
+        "query-string": "^7.1.1",
         "react-markdown": "^7.1.0",
         "remark-gfm": "^3.0.1",
         "rxjs": "^7.5.5",
         "url-loader": "^4.1.1",
+        "use-query-params": "^1.2.3",
         "use-react-router-breadcrumbs": "2.0.2"
     },
     "devDependencies": {
+        "@types/react-router-dom": "^5.3.3",
```

Comment:
This change caused typescript to fall over with errors on my machine for places we are using `useParams()`. Could you spend a few minutes to update the typings for this?

![image](https://user-images.githubusercontent.com/102054/174614999-6c57a8fb-16da-405d-94d6-9c6b580aa088.png)